chat-plugins/info.js

```diff
@@ -1357,6 +1357,7 @@ exports.commands = {
 			"- /tour create <em>format</em>, roundrobin: Creates a new round robin tournament in the current room.<br />" +
 			"- /tour end: Forcibly ends the tournament in the current room<br />" +
 			"- /tour start: Starts the tournament in the current room<br />" +
+			"- /tour banlist [pokemon], [talent], [...]: After the tournament create, banlist move, ability, pokemon, item." +
```

Comment:
This kind of sounds like a fragment to me.  How about something like this?

`"After the tournament is created, this will either ban a move, ability, pokemon, or item from being used in the tournament.<br />`

^It doesn't have to be that _exactly_, but maybe just something that reads a little better.

Also, please note the missing ``<br />`` tag at the end here.